server/controller/src/main/java/ai/starwhale/mlops/datastore/ColumnTypeScalar.java

```diff
@@ -153,7 +153,8 @@ public Object encode(Object value, boolean rawResult) {
             } else if (this == STRING) {
                 return value;
             } else if (this == BYTES) {
-                return Base64.getEncoder().encodeToString(((ByteBuffer) value).array());
+                var base64 = Base64.getEncoder().encode(((ByteBuffer) value).duplicate());
+                return StandardCharsets.UTF_8.decode(base64).toString();
```

Comment:
should there be a corresponding change in the `ColumnTypeScalar.decode` method. Base64.encoder uses `StandardCharsets.ISO_8859_1` as default charset.

Reply:
Not a problem. base64 encoded characters are all ASCII and are exactly the same in ISO_8859_1 and UTF_8.